scipy/integrate/_quadpack_py.py

```diff
@@ -615,11 +617,33 @@ def dblquad(func, a, b, gfun, hfun, args=(), epsabs=1.49e-8, epsrel=1.49e-8):
         an accuracy of ``abs(i-result) <= max(epsabs, epsrel*abs(i))``
         where ``i`` = inner integral of ``func(y, x)`` from ``gfun(x)``
         to ``hfun(x)``, and ``result`` is the numerical approximation.
-        See `epsrel` below.
+        See `epsrel` below. This arugment is deprecated and will be
+        removed in SciPy 1.11.0. Please use the `opts` argument instead.
     epsrel : float, optional
         Relative tolerance of the inner 1-D integrals. Default is 1.49e-8.
         If ``epsabs <= 0``, `epsrel` must be greater than both 5e-29
-        and ``50 * (machine epsilon)``. See `epsabs` above.
+        and ``50 * (machine epsilon)``. See `epsabs` above. This arugment is
+        deprecated instead and will be removed in SciPy 1.11.0. Please use the
+        `opts` argument instead.
+    opts : iterable object or dict, optional
+        Options to be passed to `quad`. May be empty, a dict, or
+        a sequence of dicts or functions that return a dict. If empty, the
+        default options from scipy.integrate.quad are used. If a dict, the same
+        options are used for all levels of integraion. If a sequence, then each
+        element of the sequence corresponds to a particular integration. e.g.,
+        ``opts[0]`` corresponds to integration over ``x0``, and so on. If a
+        callable, the signature must be the same as for ``ranges``. The
```

Comment:
```suggestion
        options are used for all levels of integration. If a sequence, then
        each element of the sequence corresponds to a particular integration. 
        e.g., ``opts[0]`` corresponds to integration over ``x0``, and so on. If
        a callable, the signature must be the same as for ``ranges``. The
```

Reply:
Also, `x0` is not defined AFAICT, so consider describing what it is supposed to be rather than having the reader interpret the symbol.

Reply:
It's supposed to refer to the level of integration so ``opts[i]`` corresponds to ``x_i``. I copied it from the existing documentation in ``nquad``. If we were to change it we should probably change it everywhere.

Reply:
`x0` _is_ defined in `nquad`, not here. Documentation is not guaranteed to make sense if it is copied from another function. The same is true of the documentation about callables (but I think this can just be omitted until the `epsabs` and `epsrel` are removed, because it complicates things).